_posts/en/newsletters/2020-08-05-newsletter.md

```diff
@@ -0,0 +1,303 @@
+  mempool's outpoint map.
+
+    When we receive an orphan transaction, we place it in a temporary data
+    structure called the orphan set. We then ask the peer that sent us the
+    orphan to also send us the parent transactions that we don't yet have. We can
+    do that because the orphan transaction contains the txids of its parent
+    transactions. We simply send a `getdata` message containing those txids to
+    the peer to request the parent transactions.
+
+    For [wtxid relay peers][news108 wtxid relay], transactions are announced
+    and requested using the _wtxid_ of the transaction, not the _txid_. However,
+    orphan transactions contain their parents' txids, not wtxids, so it's
+    not possible to request the parent transaction using wtxid. [PR
+    #18044][Bitcoin Core #18044], which introduced wtxid relay peers and was
+    merged last week, did not permit fetching parent transactions from wtxid
+    peers. This follow-up PR allows us to fetch those parents using the txid.
+
+    Fetching parent transactions using txid may eventually be replaced
+    by a [package relay][topic package relay] mechanism, where we can
+    ask a peer for all the unconfirmed ancestors of a transaction directly.
+
+- [Eclair #1491][] adds partial support for creating, using, and closing
+  channels that use [anchor outputs][topic anchor outputs] to both reduce
+  onchain transaction fees in normal cases and increase fees when
+  necessary for security.  This implementation handles all the basics
+  but does not yet support mutual channel closes or actual fee bumping;
+  those will be added later.  Additionally, interoperability testing
+  with LND's implementation revealed a case when the
+  [specification][anchor spec discuss] should be clarified.
+
+- [LND #4488][] updates the minimum CLTV expiry delta users may set to
+  18 blocks in line with an [updated recommendation][BOLTs #785].  The
+  default remains at 40 blocks.  When there are only this many blocks
+  until an LN payment has to be settled, LND will unilaterally close the
+  channel to ensure the latest state gets recorded onchain.  However,
+  the higher the expiry is, the more time a payment could become
+  temporarily stuck in a channel (either by accident or deliberately).
+  This has led some LN implementations to use route-finding algorithms
+  that optimize for routes with low CLTV expiry deltas, which has in
+  turn led some users to set their deltas to values that are especially
+  unsafe.  This new minimum should help prevent inexperienced users from
+  naively setting an unsafe value.
+
+- [BIPs #948][] updates the [BIP174][] specification of [PSBT][topic psbt] input
+  records to explicitly allow for providing both a non-witness UTXO (the full
+  transaction) and a witness UTXO for a single input. This is in line with the
+  [current behaviour][news105 fee overpayment] in Bitcoin Core and was motivated
```

Comment:
nit:
I appreciate that we accommodate different forms of spelling (such as this instance of behaviour), but I think it makes sense to [pick a version](https://github.com/bitcoinops/bitcoinops.github.io/pull/445/files#diff-a0054e87e2124b1941ad3d7193b79ddaR169) of 🇺🇸 vs. 🇬🇧 for same word appearing differently in the same newsletter?

Reply:
Our style guide states that "The author of the document gets to choose its flavor of English (e.g. American or British)." and the aim is to always be internally consistent within a single document, so this should be changed to behavior.

Reply:
🇺🇸 🇺🇸 🇺🇸